dcos/nodeutil/util.go

```diff
@@ -30,7 +31,7 @@ type NodeInfo interface {
 	DetectIP() (net.IP, error)
 	Role() (string, error)
 	IsLeader() (bool, error)
-	MesosID() (string, error)
+	MesosID(*context.Context) (string, error)
```

Comment:
`Context` is an interface, you don't need a pointer to it unless you intend to modify the thing that's being pointed to (which is not the case here)